tutorials/Tutorial5_Evaluation.py

```diff
@@ -106,11 +132,52 @@ def tutorial5_evaluation():
         print("Reader F1-Score:", reader_eval_results["f1"])
 
 
-    # Evaluate combination of Reader and Retriever through Finder
-    if eval_both:
-        finder_eval_results = finder.eval(top_k_retriever=1, top_k_reader=10, label_index=label_index, doc_index=doc_index)
-        finder.print_eval_results(finder_eval_results)
+    # Evaluate combination of Reader and Retriever in open domain fashion
+    elif style == "retriever_reader_open":
+
+        # Here is the pipeline definition
+        p = Pipeline()
+        p.add_node(component=retriever, name="ESRetriever", inputs=["Query"])
+        p.add_node(component=eval_retriever, name="EvalRetriever", inputs=["ESRetriever"])
+        p.add_node(component=reader, name="QAReader", inputs=["EvalRetriever"])
+        p.add_node(component=eval_reader, name="EvalReader", inputs=["QAReader"])
+        results = []
+
+        for q, l in q_to_l_dict.items():
+            res = p.run(
+                query=q,
+                top_k_retriever=10,
+                labels=l,
+                top_k_reader=10,
+                index=doc_index,
+            )
+            results.append(res)
+
+        eval_retriever.print()
+        print()
+        retriever.print_time()
+        print()
+        eval_reader.print(mode="reader")
+        print()
+        reader.print_time()
+        print()
+        eval_reader.print(mode="pipeline")
+
+
+def launch_es():
```

Comment:
I like the idea of making a function out of it. 
Cant we put this into a utils script. We should use it in the other as well